about.html

```diff
@@ -0,0 +1,71 @@
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Conference</title>
+</head>
+
+<body>
+
+    <body>
+        <section class="navbar">
+            <div class="menu-icon">
+                <hr class="m-menu">
+                <hr class="m-menu">
+                <hr class="m-menu">
+            </div>
+        </section>
+        <ul class="menu-ul">
+            <img class="close" alt="close" src="./Images/close.png">
+            <li class="links link1"><a class="menu-links" href="#home">Home</a></li>
+            <li class="links link2"><a class="menu-links" href="./about.html">About</a></li>
+            <li class="links link3"><a class="menu-links" href="#program">program</a></li>
+        </ul>
+        <section class="pg1-sec">
+            <h2 class="hello-world">"Hello, Sharing world"</h2>
+            <h1 class="event-name">The World Games <br>2022 Birmingham, USA</h1>
+            <p class="event-dis">international multi-sport event organized with the support of the International Olympic
+                Committee. Held the year following the Summer Olympic Games, The World Games 2022 will showcase a New
+                Generation of Global Sports in Birmingham from July 7-17, 2022. An anticipated 3,600 elite athletes from
+                over 100 countries will compete for gold in more than 30 of the fastest growing sports in the world. An
+                anticipated 3,600 elite athletes
+                from over 100 countries will compete for gold in more than 30 of the fastest growing sports in the
+                world.</p>
+            <p class="event-dis enjoy">Buy ticket and enjoy</p>
+            <p class="event-start">Birmingham, USA | July 7 - 17, 2022</p>
+        </section>
+        <section class="sec-logo">
+            <h2 class="what-logo">The World Games logo</h2>
+            <hr class="logo-hr">
+            <p class="dis-logo">The logo of World Games submmit 2019 was decided through the logo competition from 9
+                June to 12 Nov</p>
+            <img alt="logo" class="com-logo" src="./Images/logi.png">
+        </section>
+        <hr class="line">
+        <section class="red-img">
+            <img class="img1" src="./Images/2.jpg" alt="Evint Image">
+            <img class="img1" src="./Images/1.jpg" alt="Evint Image">
+        </section>
+        <section class="sec-logo partners">
+            <h3 class="what-logo">Partner</h3>
+            <hr class="logo-hr">
+            <div class="partner-logo-container">
+                <img alt="Microsoft logo" class="patner-logo" src="./Images/microsoft.png">
+                <img alt="Coca Cola logo" class="patner2-logo" src="./Images/cocacola.png">
+            </div>
+        </section>
+        <section class="sec-logo last-part">
+            <p class="dis-logo last-p">The World Game <br> Some Rights Reserved.</p>
+        </section>
```

Comment:
- [ ] Consider your footer contains a logo aligned to the left and text aligned to the right.